Examples/Example23-Charts/Example-ChartTimeLine02.ps1

```diff
@@ -0,0 +1,42 @@
+﻿Import-Module .\PSWriteHTML.psd1 -Force
+
+New-HTML -TitleText 'Charts - TimeLine' -Online -FilePath $PSScriptRoot\Example-ChartTimeLine.html {
+    New-HTMLSection -Invisible {
+        New-HTMLPanel {
+            New-HTMLChart -Title 'Incidents Reported vs Solved' -TitleAlignment center {
+                New-ChartTooltip -TitleText 'Timeline' -XAxisFormatPattern 'HH:mm:s'
+                New-ChartAxisX -MinValue ((Get-Date).AddDays(-1)) -MaxValue ((Get-Date).AddDays(1)) -Type 'datetime'
+                $From = (Get-Date).AddDays(-1)
+                $To = (Get-Date)
+                New-ChartTimeLine -DateFrom $From -DateTo $To -Name 'Test 0'
+                $From = (Get-Date).AddHours(-1)
+                $To = (Get-Date)
+                New-ChartTimeLine -DateFrom $From -DateTo $To -Name 'Test 1' -Color Red
+                $From = (Get-Date).AddDays(0)
+                $To = (Get-Date).AddHours(5)
+                New-ChartTimeLine -DateFrom $From -DateTo $To -Name 'Test 2' -Color Yellow
+                $From = (Get-Date).AddHours(-5)
+                $To = (Get-Date).AddHours(11)
+                New-ChartTimeLine -DateFrom $From -DateTo $To -Name 'Test 3' -Color Beige
+                $From = (Get-Date).AddMinutes(15)
+                $To = (Get-Date).AddHours(3)
+                New-ChartTimeLine -DateFrom $From -DateTo $To -Name 'Test 4'
+            }
+        }
+        New-HTMLPanel {
+            New-HTMLChart -Title 'Incidents Reported vs Solved' -TitleAlignment center {
+                New-ChartTooltip -TitleText 'Timeline' -XAxisFormatPattern 'HH:mm:s'
```

Comment:
Shouldn't it be

'HH:mm:**ss**'

Reply:
Probably yes :-)

Reply:
![image](https://user-images.githubusercontent.com/15063294/99369991-6bf3c580-28bd-11eb-9a03-f2613f40f5c0.png)

But it works regardless